js/autotrack/textInput.js

```diff
@@ -0,0 +1,37 @@
+import { getBaseComponentProps } from './common';
+import * as _ from 'lodash';
+
+export const autocaptureTextInputChange = track => (
+  eventType,
+  componentThis,
+  event
+) => {
+  // Attach a debounce function to the TextInput component instance if one's not already attached.
+  if (!componentThis.__heap__debounceTextChange) {
+    componentThis.__heap__debounceTextChange = _.debounce(
+      debouncedAutocaptureTextInputChange(track),
+      1000
+    );
+  }
+
+  componentThis.__heap__debounceTextChange(eventType, componentThis, event);
+};
+
+const debouncedAutocaptureTextInputChange = track => (
+  eventType,
```

Comment:
It might be helpful to add a comment to link to the event types that we're expecting (could just be a link to RN source on github or a doc if they have it)

Reply:
In this case, the event type is included as a string literal that we specify in the instrumentation code: https://github.com/heap/react-native-heap/pull/129/files#diff-df6ad962b9c024cab3a8be043f101c81R142